.github/workflows/ci.yaml

```diff
@@ -0,0 +1,47 @@
+name: "CI"
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: "0 7 * * *"
```

Comment:
I'm not super convinced into the need for a daily cron job, mostly because upstream never really changes that quickly. How would we feel about a twice weekly cron instead?